pkg/cmd/root/root.go

```diff
@@ -43,6 +43,7 @@ var (
 		gf.ConfigFileFlag:       cfg.GetDefaultCfgFileName(),
 		gf.MockDelayInSecFlag:   0,
 		gf.MockTriggerTimeFlag:  "",
+		gf.TerminationNodesFlag: 99,
```

Comment:
do you think `-1` would be more intuitive to be unlimited?  99 seems pretty arbitrary.   

I'm also not sure we should default to unlimited. I understand why you did, since it maintains backwards compatibility. It may also be difficult to find a good number that doesn't surprise some users, "Why did some of my nodes not receive ITNs?"  But I think that's a better problem than causing a whole cluster to die.   What would you think about setting it at 2?

Reply:
* +1 to the `-1` recommendation
* I'm good with defaulting to `2`
  * do you think I should default this in the helm `values.yaml` file as well even though it'll be defaulted in app code? Not sure the etiquette on that
  * since this will technically break backwards compat should we major release it?